docs/reference/api/platform/MemoryStats.md

```diff
@@ -1,11 +1,15 @@
 ## MemoryStats
 
-[Add description here.]
+Mbed OS provides a set of functions that you can use to capture the memory statistics at runtime. `mbed_stats.h` declares these functions. You can use memory statistics functions to capture heap usage, cumulative stack usage or stack usage per thread at runtime. To enable memory usage monitoring, you must build Mbed OS with the following macros.
```

Comment:
There are some differences between tracing and stats. It would be good to call them out explicitly and explain how each would be used.

Reply:
Good catch, actually I shouldn't have mentioned MBED_MEM_TRACING_ENABLED at all. There is a separate requirement for writing "mem_trace functions" assigned to Martin. So let me clean this up and remove the references to MBED_MEM_TRACING_ENABLED. I think it should be captured in mem_trace docs, not here.